finance-tracker/app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
What does this have to do with adding bootstrap as per your PR title suggest?

Reply:
this is bootstrap's navbar code